### ISSUE TITLE
[Bug]: T8210 Battery Doorbell - Battery Status Incorrect

### ISSUE THREAD

**Issue by BadgerLoaf** [bropat/eufy-security-ws#154]
### Version

v2.8.1  fuatakgun / eufy_security

### Node version

Core Version 2022.11.1

### Operating System type

Linux

### Operating system version

Home Assistant OS 9.3

### Describe the bug

The battery level of my T8210 battery doorbell is not being updated, and appears to remain static. It does not reflect the battery level shown within Eufy phone app. Current the HA battery sensor states 85% for the doorbell, but the phone app states 63%.

This appears to be the same symptoms detailed in Issue #57, though that looks like it was resolved?

 

### To reproduce

View `sensor.doorbell_battery`, this shows a different value to that showing in the Eufy phone app. This value doesn't appear to change, or at least changes very infrequently. 

### Screenshots & Logfiles

Happy to provide logs etc, just let me know what you need.

### Additional context

Other functions of the doorbell seem to work fine in HA - I can stream video etc.

**Comment by derekoharrow** [comment]
I am experiencing the same with both the battery doorbell and with battery external cameras. Camera is down to 14%, but this is reporting 47% and has done for many days now

**Comment by cclarke591** [comment]
I've run into the same issue. Though it seems like once a day it will update. I have also tried force sync, but that doesn't seem to help. I did find that removing the add-on and re-adding it, along with the doorbell, it then pulled the latest battery.

**Comment by BadgerLoaf** [comment]
@bropat Is this something you might be able to look at? Happy to test the issue and provide logs as required.

**Comment by scsa20** [comment]
Same thing happens with the eufyCam 2C, been like this since I first got it configured about 8 months ago, but it just doesn't update for when the cameras are charged (it only updates when it's discharging) until I go to the Devices and Services and reload the Eufy Security Camera integration (I've even tried a force sync but no go, only reloading the integration gets it to show the updated battery status of it being charged).

**Comment by BadgerLoaf** [comment]
I have updated the version info on the initial post, as I have tested against the latest 1.3.0 version, and the issue still remains.

Any thoughts @bropat ? Thanks!

Current Log contents:
`2023-01-05 11:08:43.308  INFO  Eufy Security server listening on host 0.0.0.0, port 3000 
2023-01-05 11:08:53.966  INFO  Push notification connection successfully established 
2023-01-05 11:08:54.052  INFO  Connected to station T8010P232102139F on host 10.10.20.53 and port 17028 
2023-01-05 11:08:54.070  INFO  Connected to station T8400P2020360EF5 on host 10.10.20.91 and port 28712 
2023-01-05 11:08:54.093  INFO  Connected to station T8400P2020360EE2 on host 10.10.20.59 and port 29322 
2023-01-05 11:09:03.709  INFO  Connected to station T8400P2020362521 on host 10.10.20.93 and port 23682 
2023-01-05 11:09:57.924  INFO  Disconnected from station T8010P232102139F 
2023-01-05 11:10:03.023  INFO  Connected to station T8010P232102139F on host 10.10.20.53 and port 28902 
2023-01-05 11:11:06.880  INFO  Disconnected from station T8010P232102139F 
2023-01-05 11:11:11.974  INFO  Connected to station T8010P232102139F on host 10.10.20.53 and port 27428`

**Comment by WillWNL** [comment]
> I have updated the version info on the initial post, as I have tested against the latest 1.3.0 version, and the issue still remains.
> 
> Any thoughts @bropat ? Thanks!
> 
> Current Log contents: `2023-01-05 11:08:43.308 INFO Eufy Security server listening on host 0.0.0.0, port 3000 2023-01-05 11:08:53.966 INFO Push notification connection successfully established 2023-01-05 11:08:54.052 INFO Connected to station T8010P232102139F on host 10.10.20.53 and port 17028 2023-01-05 11:08:54.070 INFO Connected to station T8400P2020360EF5 on host 10.10.20.91 and port 28712 2023-01-05 11:08:54.093 INFO Connected to station T8400P2020360EE2 on host 10.10.20.59 and port 29322 2023-01-05 11:09:03.709 INFO Connected to station T8400P2020362521 on host 10.10.20.93 and port 23682 2023-01-05 11:09:57.924 INFO Disconnected from station T8010P232102139F 2023-01-05 11:10:03.023 INFO Connected to station T8010P232102139F on host 10.10.20.53 and port 28902 2023-01-05 11:11:06.880 INFO Disconnected from station T8010P232102139F 2023-01-05 11:11:11.974 INFO Connected to station T8010P232102139F on host 10.10.20.53 and port 27428`

Same as #174

**Comment by derekoharrow** [comment]
Beta v6 seems to be much better

**Comment by fuatakgun** [comment]
@derekoharrow , v6 is about eufy_security_integration, eufy-security-ws is on v.1.3.

So, they are not one to one related

**Comment by BadgerLoaf** [comment]
Agreed - the issue started with not getting Battery Status, but now it seems that I am getting the same disconnect messages as #174... possibly connected or not to the original battery status issue


> Same as #174

**Comment by inertia666** [comment]
Same problem on T8220. I am not sure the integration is working very well on the whole though.

**Comment by kloodhu** [comment]
> 

i think the same. the whole thing is very unstable. connect, disconnect, zero data, nothing changes,...
i need to restart the addon in hass, then reload the integration to get new data.

**Comment by dumbo0001** [comment]
Is there an indication what is causing this problem and if it is fixable? Because the battery drains fast it is good to have notification when battery is low.

**Comment by benandez** [comment]
[home-assistant_eufy_security_2023-06-10T06-55-14.448Z.log](https://github.com/bropat/eufy-security-ws/files/11711797/home-assistant_eufy_security_2023-06-10T06-55-14.448Z.log)

I have the same issue with a new install of the Eufy integration. I am quite new to Home Assistant so still figuring a lot out, hopefully the logs will help you :)

**Comment by ersan** [comment]
Also having this issue with 2C cameras, I think it's because Eufy rarely updates the battery level in their cloud API and you have to retrieve it directly from the Homebase to get an accurate measurement.

**Comment by bropat** [comment]
Please use the latest version (`1.6.3`). If the problem persists, please open a new issue with the latest log.
